### PR TITLE
Add trace for WC SRTServletRequestPart

### PR DESCRIPTION
**Pull request by pmd1nh**
Add trace for SRTServletRequestPart to help debugging FileUpload

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build
        #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Review by isaacrivriv — APPROVED** [review]
Additional logging looks good. Thanks for working on this!

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java [hunk 1]
```diff
@@ -18,68 +18,87 @@
...
+    protected static final Logger logger = LoggerFactory.getInstance().getLogger("com.ibm.ws.webcontainer.srt");
+
+
+    public SRTServletRequestPart(DiskFileItem item) {
+        _part = item;
+    }
+    @Override
+    public void delete() throws IOException {
+        if (System.getSecurityManager() != null) {
+            if (TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)) {
```
Comment (volosied) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java:47]: Spacing around && for these new log statements, if you wanted to update it.

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java [hunk 2]
```diff
@@ -134,13 +164,13 @@ public Object run() {
                 headersList.add((String)it.next());
             }
         }
-		return headersList;
-	}
+        return headersList;
```
Comment (volosied) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java:167]: Should this have any return value logging?

Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java:167]: probably not

Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestPart.java:167]: prbably not

**Comment by pmd1nh** [comment]
unrelated failed tests.

**Pull request merged**